editor/code_editor.cpp

```diff
@@ -1572,20 +1572,20 @@ void CodeTextEditor::goto_line(int p_line) {
 	text_editor->remove_secondary_carets();
 	text_editor->deselect();
 	text_editor->unfold_line(p_line);
-	text_editor->call_deferred(SNAME("set_caret_line"), p_line);
+	callable_mp((TextEdit *)text_editor, &TextEdit::set_caret_line).call_deferred(p_line);
```

Comment:
When using the Script editor's member overview to go to a method, it doesn't work and I get `core\object\message_queue.cpp:223 - Error calling deferred method: 'CodeEdit::TextEdit::set_caret_line': Method expected 5 arguments, but called with 1.`
Do all the arguments need to be explicitly passed now?
In that case this should be ...`call_deferred(p_line, true, true, 0, 0);`.

Reply:
It always has to, see above, this is a missed case of default arguments

Reply:
```suggestion
	callable_mp((TextEdit *)text_editor, &TextEdit::set_caret_line).call_deferred(p_line, true, true, 0, 0);
```